src/Samples/DownloadFile/DownloadFile.targets

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I'd say to not specify the xml header or the namespace in samples, since they're no longer needed.